homeassistant/components/advantage_air/__init__.py

```diff
@@ -69,8 +70,8 @@ async def error_handle(param):
     hass.data.setdefault(DOMAIN, {})
     hass.data[DOMAIN][entry.entry_id] = {
```

Comment:
For future improvement, you could add a named tuple or data class to hold these items instead.